zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -271,11 +278,32 @@ benchmarks! {
     }: { call.dispatch_bypass_filter(destroy_origin)? }
 
     admin_move_market_to_closed {
```

Comment:
when closing market, its relevant pool also gets closed. Should these benchmarks also consider that? or it is not a very complex operation and thus  won't impact weight much.